githubteacher-orlando.md

```diff
@@ -0,0 +1,8 @@
+# Orlando, FL
+
+### Restaurants
```

Comment:
I like [Pharmacy](http://thepharmacyorlando.com) in Dr. Philips and [Highball & Harvest](http://www.ritzcarlton.com/en/hotels/florida/orlando/dining/highball-and-harvest) at The Ritz.

Reply:
I like 🍣, and deep dish 🍕

Reply:
And 🍺? How about 🍻?

Reply:
@hectorsector Added some restaurants, let me know this if this is good to :ship: